src/test/java/io/camunda/zeebe/bpmnassert/testengine/db/InMemoryZeebeDbTransactionTest.java

```diff
@@ -468,6 +468,27 @@ public void shouldNotCommitOnError() {
     assertThat(threeColumnFamily.exists(threeKey)).isFalse();
   }
 
+  @Test
+  public void shouldWriteKeyAfterDeletion() {
```

Comment:
:wrench: If you have time please, also create a PR in EZE repository. I would like to give back some code improvements where possible, as a thank you for letting us copy their stuff.